content/en/observability_pipelines/setup/_index.md

```diff
@@ -5,8 +5,399 @@ type: multi-code-lang
 aliases:
   - /getting_started/observability_pipelines/
   - /observability_pipelines/installation/
+further_reading:
+  - link: "/observability_pipelines/working_with_data/"
+    tag: "Documentation"
+    text: "Working with data in Observability Pipelines"
+  - link: "/observability_pipelines/production_deployment_overview/"
+    tag: "Documentation"
+    text: "Deployment Design and Principles for the Observability Pipelines Worker"
+  - link: "/observability_pipelines/architecture/"
+    tag: "Documentation"
+    text: "Production deployment design and principles for the Observability Pipelines Worker"
+  - link: "https://dtdg.co/d22op"
+    tag: "Learning Center"
+    text: "Safe and Secure Local Processing with Observability Pipelines"
 ---
 
-For setup instructions, select your use case:
+## Overview
+
+The [Observability Pipelines Worker][1] can collect, process, and route logs and metrics from any source to any destination. Using Datadog, you can build and manage all of your Observability Pipelines Worker deployments at scale.
+
+There are several ways to get started with the Observability Pipelines Worker.
+
+- [Quickstart](#quickstart): Install the Worker and test out the functionality of Observability Pipelines with demo data.
+- [Datadog setup guide][2]: Install the Worker and connect it to Datadog.
+- [Splunk setup guide][3]: Install the Worker and connect it to Splunk.
```

Comment:
"Splunk setup guide: Install the Worker with an out-of-the-box pipeline for receiving and routing data from Splunk HEC to both Splunk and Datadog"